query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/CswProfile.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.sources.fields;
+
+import org.codice.ddf.admin.api.fields.Field;
+import org.codice.ddf.admin.common.fields.base.BaseEnumField;
+import org.codice.ddf.admin.common.fields.base.scalar.StringField;
+
+import com.google.common.collect.ImmutableList;
+
+public class CswProfile extends BaseEnumField<String> {
+
+    public static final String DEFAULT_FIELD_NAME = "cswProfile";
+
+    public static final String TYPE_NAME = "CswProfile";
+
+    public static final String DESCRIPTION =
+            "CSW application profile specifying the capabilities of the CSW server when federating to other systems.";
+
+    public static final String CSW_SPEC_PROFILE_FEDERATED_SOURCE =
```

Comment:
Changed these to `CswFederatedSource`, `DDFCswSource`, and `GmdCswFederatedSource`